core/src/broadcast_stage.rs

```diff
@@ -190,7 +190,7 @@ impl BroadcastStage {
             let mut broadcast_table = cluster_info
                 .read()
                 .unwrap()
-                .sorted_tvu_peers(&staking_utils::node_stakes(&bank));
+                .sorted_tvu_peers(&staking_utils::delegate_stakes(&bank));
```

Comment:
Maybe it's just me, but it sounds like this is gonna delegate some stakes. :) @garious?

Reply:
yep, but `delegated_stakes` would be fine